### DISCUSSION TITLE
Broken

### DISCUSSION THREAD

**Discussion by Marcowar1985** [tingjoybits/Brush_Manager#10] [General]
I really like this addon, it's a shame it's not getting any love now. 
It still works in blender 3.6 LTS, but missing options. 
needs updated pls!

**Comment by tingjoybits** [comment]
Hello, glad you liked the addon.  It seems I didn't mention that the development of the addon is temporarily suspended due to the war in my country, so my free time is extremely limited.  Recently I have been making edits to support Blender 4.1, if there are errors, it would be nice for you to describe more specifically which ones.  Regarding the need for options, I'm not sure what you mean.  Can you describe what you would like to update?

**Comment by Marcowar1985** [comment]
100% a valid reason.
I won't push, I hope everything works out for the best for everyone.
________________________________

**Comment by Marcowar1985** [comment]
If you are interested, the problem is missing the drop down tab.
[cid:dd7b0ce7-46cb-4073-b1a9-777d82566006][cid:7e2191c6-82c3-4c3f-9e81-940bfc38f41c]
You lose a lot of functionality. I'm trying to Chat GTP a fix but the script is too large for it to process haha!

________________________________

**Comment by tingjoybits** [comment]
What version of blender does this problem occur in?  I will try to fix it as soon as possible.

**Comment by Marcowar1985** [comment]
Blender 3.6.10LTS
________________________________
